woocommerce/woocommerce-gutenberg-products-block
# Product Gallery Block: Add zoom on hovering on the frontend - E2E tests

## What

This PR adds the E2E test for #9941.

## Why



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Ensure that the E2E tests pass on the CI.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
